scripts/environment/bootstrap-macos-10.sh

```diff
@@ -6,5 +6,5 @@ brew install ruby coreutils cuelang/tap/cue
 echo "export PATH=\"/usr/local/opt/ruby/bin:\$PATH\"" >> "$HOME/.bash_profile"
 
 if [ -n "${CI-}" ] ; then
-  echo "::add-path::/usr/local/opt/ruby/bin"
+  echo "/usr/local/opt/ruby/bin" >> "$GITHUB_PATH"
```

Comment:
Interesting, does this work? I didn't encounter it in the docs before...

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files

Reply:
Nice, thx!